hybrasyl/Utility.cs

```diff
@@ -280,6 +281,116 @@ static class LogLevels
         public const int DEBUG = 7;
     }
 
+    class GameFolders
+    {
+        private static GameFolders instance;
+
+        private GameFolders() {}
+
+        public static GameFolders Instance

```

Comment:
I don't think this is needed - in my mind there is no reason for this to be a instantiable class instead of static.

Reply:
Fixed